ui/apps/platform/src/Containers/Login/LoginPage.js

```diff
@@ -306,7 +319,7 @@ const LoadingOrForm = ({ authProviders, authorizeRoxctlMode = false }) => {
     }
 
     const options = authProvidersToSelectOptions(availableAuthProviders);
-    const initialValues = { authProvider: options[0].value };
+    const initialValues = { authProvider: options[0]?.value };
```

Comment:
In contrast to the other comment, this makes sense.

How about a comment to explain when the array might be empty?